subworkflows/ebi-metagenomics/fasta_domainannotation/main.nf

```diff
@@ -0,0 +1,45 @@
+include { BLAST_MAKEBLASTDB } from '../../../modules/ebi-metagenomics/blast/makeblastdb/main'
+include { BLAST_BLASTP      } from '../../../modules/ebi-metagenomics/blast/blastp/main'
+include { DIAMOND_MAKEDB    } from '../../../modules/ebi-metagenomics/diamond/makedb/main'
+include { DIAMOND_BLASTP    } from '../../../modules/ebi-metagenomics/diamond/blastp/main'
+include { INTERPROSCAN      } from '../../../modules/ebi-metagenomics/interproscan/main'
+
+workflow FASTA_DOMAINANNOTATION {
```

Comment:
```suggestion
workflow FASTA_DOMAIN_ANNOTATION {
```

Reply:
Keeping it as is for the time being. Will probably be changed by the nf-core community based on their naming conventions.